gcovr/formats/html/github/templates/directory_page.summary.html

```diff
@@ -11,48 +11,75 @@
                 <span>{{info.get_directory()}}</span>
             </div>
             <div>Date: {{info.date}}</div>
-            <div>
-                <span class="mr-2">Coverage:</span>
-                <span class="IssueLabel coverage-low color-fg-on-emphasis mr-1">low: &ge; 0%</span>
+            <div class="d-flex flex-wrap" style="margin-bottom: 2px;">
+                <div class="mr-2" style="min-width: 65px;">Coverage:</div>
```

Comment:
The Lines row in https://bz17.dev/gcovr/legends/ is 65px long the other ones are about 67px on my screen. Maybe this here needs to be increased.

Reply:
Pushed.
Yes you are right in my screen is almost 66px not 65px. I 've increased min-width to 67px.

Reply:
On my screen it was 67,067. I think 70 will be better.

Reply:
Changed